composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt

```diff
@@ -99,8 +103,14 @@ fun Navigation(
         ) { entry ->
             val category = entry.arguments?.getString("category") ?: return@composable
             when (category) {
-                PreferenceCategoryKey.SEND_EMAIL.name -> {
-                    // TODO: Implement based on platform
+                PreferenceCategoryKey.ABOUT_OONI.name -> {
+                    val viewModel =
```

Comment:
In these "dumb" views like the About, where no state is read, I'm OK with not having a ViewModel. But it's your call 😉

Reply:
I will leave the `viewModel` provides an organized way of event handling from the view